.gitignore

```diff
@@ -100,13 +100,15 @@ test/sequence
 test/grid
 test/gridded_data
 test/location
+test/access_restrictions
```

Comment:
test binaries will be placed to `<build dir>/unit_tests/` directory that is specified by [`RUNTIME_OUTPUT_DIRECTORY`](https://github.com/valhalla/valhalla/blob/b860d71c2581ff9a5dd7af97b03e599ec7ba4274/CMakeLists.txt#L194) target property.

Reply:
yes, but it is better to clean .gitignore file after finalizing cmake builds.